lib/clients/rtm/client.js

```diff
@@ -47,8 +47,6 @@ var wsSocketFn = require('../transports/ws');
  * @param {boolean} opts.autoReconnect Whether or not to automatically reconnect when the connection
  *     closes.
  * @param {boolean} opts.useRtmConnect True to use rtm.connect rather than rtm.start
- * @param {number} opts.maxReconnectionAttempts
```

Comment:
one concern here: removing these options from the API is backwards-incompatible. can we give some thought to how we might be able to turn these options into a parameterized retry policy on the fly?